BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -111,27 +111,47 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
     
     walletView.headerView.layoutIfNeeded()
     walletView.contentView?.layoutIfNeeded()
+    walletView.rewardsSummaryView.layoutIfNeeded()
     
     guard let contentView = walletView.contentView else { return }
-    
-    var height: CGFloat = walletView.headerView.bounds.height + walletView.rewardsSummaryView.rewardsSummaryButton.bounds.height
-    if let scrollView = walletView.contentView?.innerScrollView {
-      scrollView.contentInset = UIEdgeInsets(top: walletView.headerView.bounds.height, left: 0, bottom: 0, right: 0)
-      scrollView.scrollIndicatorInsets = scrollView.contentInset
-      
-      height += scrollView.contentSize.height
-    } else {
-      height += contentView.bounds.height
-    }
-    
+    var height: CGFloat
     if state.ledger.isEnabled {
-      height = RewardsUX.preferredPanelSize.height
+      if summaryRows.isEmpty && isLocal {
```

Comment:
All of this additional layout logic seems incorrect, if you add a size to the empty content view you created is this needed?